_data/contributors.yml

```diff
@@ -214,6 +214,29 @@
   email: ioana.ifrim@cern.ch
   education: MPhil Advanced Computer Science, University of Cambridge (2018)
 
+- name: Sahil Patidar
```

Comment:
This needs to go before `- name: "This could be you!"`. And also add an image.

Reply:
ping.

Reply:
@SahilPatidar, ping.

Reply:
Sorry for the delay, I was a bit swamped. I will do it today.

Reply:
We will need the new proposal with the new project title.